pkg/proxy/ipvs/proxier.go

```diff
@@ -362,10 +362,14 @@ func NewProxier(ipt utiliptables.Interface,
 		}
 	}
 
-	// Set the connection reuse mode
-	if val, _ := sysctl.GetSysctl(sysctlConnReuse); val != 0 {
-		if err := sysctl.SetSysctl(sysctlConnReuse, 0); err != nil {
-			return nil, fmt.Errorf("can't set sysctl %s: %v", sysctlConnReuse, err)
+	if kernelVersion.LessThan(version.MustParseGeneric("4.19")) {
```

Comment:
can we store the "4.19" string as a constant called `connReuseMinSupportedKernelVersion` or something similar?